find_xml_file.rb

```diff
@@ -0,0 +1,15 @@
+
+def find_xml_file(folder)
+  files = Dir.entries(folder)
+
```

Comment:
I think this code works. I wonder if there is a way to use .include in place of a .each loop unless there is a reason for choosing to use an .each loop? If so, would love to hear why for future reference!